src/libraries/System.Configuration.ConfigurationManager/tests/System/Configuration/ConfigurationPathTests.cs

```diff
@@ -0,0 +1,92 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Configuration;
+using System.IO;
+using Microsoft.DotNet.RemoteExecutor;
+using Xunit;
+
+namespace System.ConfigurationTests
+{
+    public class ConfigurationPathTests
+    {
+        private const string ConfigName = "APP_CONFIG_FILE";
```

Comment:
It might be clearer if you just inline this value. ConfigName is a bit of a vague term.